src/main/java/com/andreamazzarella/contact_manager/ConsoleRunner.java

```diff
@@ -0,0 +1,36 @@
+package com.andreamazzarella.contact_manager;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+public class ConsoleRunner {
+    public static void main(String[] args) throws IOException {
+        InMemoryRepository myRepo = new InMemoryRepository();
+
+        initialiseRepository(myRepo);
+
+        TextBasedContactManager contactManager = new TextBasedContactManager(System.in, System.out, myRepo);
+        contactManager.run();
+    }
+
+    private static void initialiseRepository(InMemoryRepository myRepo) {
+        HashMap<String, String> andreasInfo = new HashMap<>();
+        andreasInfo.put("name", "Andrea");
+        andreasInfo.put("number", "774444");
+        Contact andrea = new Contact(andreasInfo);
```

Comment:
I feel like you're losing out of Java's benefits from using a string->string hashmap for contact data. Java's type system has huge advantages and offers compile time errors when the programmer makes a mistake. I would expect Contact to have something like;

String firstname;
PhoneNumber (or integer) mobileNumber,
Date dateOfBirth;